tcpdf.php

```diff
@@ -7711,7 +7708,8 @@ public function Output($name='doc.pdf', $dest='I') {
 					header('Pragma: public');
 					header('Expires: Sat, 26 Jul 1997 05:00:00 GMT'); // Date in the past
 					header('Last-Modified: '.gmdate('D, d M Y H:i:s').' GMT');
-					header('Content-Disposition: inline; filename="'.basename($name).'"');
+					header('Content-Disposition: inline; filename="' . rawurlencode(basename($name)) . '"; ' .
+						'filename*=U-TF8\'\'' . rawurlencode(basename($name)));
```

Comment:
```suggestion
						'filename*=UTF-8\'\'' . rawurlencode(basename($name)));
```

Reply:
Wow, can't believe I missed that - thanks @Fighter456 for your quick response :+1: